Sources/PT.PM.Common/Helper.cs

```diff
@@ -52,5 +54,14 @@ public static bool IsRunningOnLinux
                 return (p == 4) || (p == 6) || (p == 128);
             }
         }
+
+        static public bool TryCheckIdTokenValue(Expression expr, string value)
+        {
+            if (expr?.NodeType != NodeType.IdToken)
+            {
+                return false;
+            }
+            return ((IdToken)expr).TextValue == value;
```

Comment:
Maybe replace with `return expr is IdToken idToken && idToken.TextValue == value`?